README.md

```diff
@@ -1,5 +1,34 @@
-A tool to dump the ROM of the Xbox DVD Movie Playback Kit dongle.
+## A tool to dump the ROM of the Xbox DVD Movie Playback Kit dongle.
 
 The ROM is prefixed with version information and the length of the contained XBE.
 The XBE certificate region is the allowed DVD playback region.
 The section of the XBE is compressed using LZX, 128kiB window size, 32kiB blocks.
+
+## Prerequisites
+
+Make sure you satisfy the following requirements:
+
+* basic python know-how
```

Comment:
This line should be removed - it's expected when people look at github.

*(On a somewhat related note: We might eventually have a CI version which is bundled into a Windows executable, but that's not relevant for now)*